recipe/meta.yaml

```diff
@@ -30,10 +30,12 @@ requirements:
 test:
   requires:
     - pytest
```

Comment:
If we don't use python 3.10 yet you can add `python <3.10` to the test/requires

Reply:
> If we don't use python 3.10 yet you can add `python <3.10` to the test/requires

The tests section was updated
https://github.com/AnacondaRecipes/pooch-feedstock/blob/update_to_1.5.2/recipe/meta.yaml#L34